frontend/src/components/TaskCalendar.jsx

```
@@ -108,13 +632,53 @@ const TaskCalendar = () => {
+    const calendarRect = calendarRef.current.getBoundingClientRect();
+    const W = calendarRect.width;
+    const H = calendarRect.height;
+
+    let safeX = 10;
+    if (x < W / 2) {
+      safeX = x + 10;
+    }
+    else {
+      safeX = x - W / 2;
+    }
+    let safeY = H / 4;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'safeY' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)